src/main/resources/application.yml

```diff
@@ -2,6 +2,9 @@ server:
   port: 8080
 logging.pattern.console: "%d{&quot;yyyy-MM-dd'T'HH:mm:ss.SSSZ&quot;} %highlight(%-5level) [%thread] rid=[%X{rid}] %logger{36} - %msg %rEx{30}%n"
 pitchfork:
+  validators:
+    accept-null-timestamps: true
+    max-timestamp-drift-seconds: 3600
```

Comment:
Since the default for the application is "unconfigured" - i.e. no forwarders enabled by default, perhaps this should be -1.

Reply:
Yeap good point.